clang/test/SemaHLSL/BuiltIns/asuint-errors.hlsl

```diff
@@ -0,0 +1,25 @@
+// RUN: %clang_cc1 -finclude-default-header -triple dxil-pc-shadermodel6.6-library %s -fnative-half-type -emit-llvm-only -disable-llvm-passes -verify
```

Comment:
The `-emit-llvm-only` and `-disable-llvm-passes` flags don't really do anything when combined with verify. We don't need them.
```suggestion
// RUN: %clang_cc1 -finclude-default-header -triple dxil-pc-shadermodel6.6-library %s -fnative-half-type -verify
```